### ISSUE TITLE
Installing mods from shared profile asks for options

### ISSUE THREAD

**Issue by prinyo** [Nexus-Mods/NMM-Test-Builds#26]
Downloaded a profile, downloaded missing mods, switched to the profile and it started installing. 3 of the mods had scripted installers and it asked me for options.

**Comment by roamwild** [comment]
Yup, some mods don't seem to get an .xml in a profile's 'Scripted' folder, haven't looked at individual archives / FOMOD configs yet but there is probably some simple reason?

e.g. http://www.nexusmods.com/fallout4/profiles/2155

No .XML in 'Scripted', asks for options on install:
4640  spring cleaning
7515  better power armor - a power armor overhaul
6209  carry more with power armor
214   carry weight modifications - FOMOD version
4042  legendary modification

Automatically installed / in scripted folder:

4772 Better Settlers
9758 Better Vendor Stalls
2494 Brighter Settlement Lights
325 Config. Fusion Core Drain
5108 keyNuker
7525 Land Purifiers
3176 MMP4APA
7393 snap n build
7399 this is my bed
5346 wedding rings

**Comment by DuskDweller** [comment]
The issue here seems to be the lack of scripted installer xml files in the Scripted folder, I'm wondering whether the profile was made using earlier versions of those mods where there was no scripted installer (eg. carry weight mod had no installer in earlier releases).

**Comment by KunoMochi** [comment]
I recently installed then uninstalled [_Blowing in the Wind_](http://www.nexusmods.com/skyrim/mods/73916/?) for Skyrim to update from version _1.3_ to _1.5_ and I encountered this as well. _1.3_ previously had a scripted installer already as does _1.5_.

I did the mod update different than I usually do. Instead of uninstalling the previous version first, I installed the up to date _1.5_ version first then uninstalled version _1.3_.

**Steps taken:**
1. Downloaded and installed _Blowing in the Wind 1.3_
2. Downloaded and installed _Blowing in the Wind 1.5_
3. Uninstalled _Blowing in the Wind 1.3_
4. Created a new profile
5. Switched back to main profile

**Comment by roamwild** [comment]
>  I'm wondering whether the profile was made using earlier versions of those mods where there was no scripted installer (eg. carry weight mod had no installer in earlier releases).

Not sure this is the case, as most of the mods have always (as far as I can tell) used FOMOD installers. Even after making a copy of the profile in my previous post, and installing the mods from scratch, these mods never had an .xml created in Scripted folder.

I also had this issue with my own local profiles.

Now for whatever reason, all mods I'm testing are getting .XML files in Scripted folder of the profile.

I created a new profile roamtest01 with 'fresh' first-time installations of all the mods (missing Scripted .xml) listed in my previous post.

> No .XML in 'Scripted', asks for options on install:
> 4640 spring cleaning
> 7515 better power armor - a power armor overhaul
> 6209 carry more with power armor
> 214 carry weight modifications - FOMOD version
> 4042 legendary modification

These all created .xml files in Scripted folder of the profile, and install properly without asking for options when the profile is activated.

**Comment by DuskDweller** [comment]
Anyone else experiencing this issue with specific mods?

**Comment by prinyo** [comment]
Maybe someone can upload a profile with several mods like this and we can download and test?
